### PR TITLE
feat: contracts list in UI and API

### LINKED ISSUE #91: UI: list latest contracts

### PR DESCRIPTION
**Pull request by FilipHarald**
resolves #91

### CODE REVIEW

File: services/explorer-ui/src/routes/contracts/index.lazy.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        blockHash: contractInstance.blockHash,
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        blockHeight: contractInstance.blockHeight,
+        version: contractInstance.version,
+        contractClassId: contractInstance.contractClassId,
+        publicKeysHash: contractInstance.publicKeysHash,
+        deployer: contractInstance.deployer,
+      })
+  );
+  let isIndex = true;
```
[the remark this answers is not in the document]
Comment (FilipHarald) [reply services/explorer-ui/src/routes/contracts/index.lazy.tsx:29]: AI told me to do it and I wanted to focus on smth else. You can remove it in your version OK?

[the remark this answers is not in the document]
Comment (Mautjee) [reply services/explorer-ui/src/routes/contracts/index.lazy.tsx:29]: hahah oke yeah I will remove it

[the remark this answers is not in the document]
Comment (Mautjee) [reply services/explorer-ui/src/routes/contracts/index.lazy.tsx:29]: maybe at a todo for me?

[the remark this answers is not in the document]
Comment (FilipHarald) [reply services/explorer-ui/src/routes/contracts/index.lazy.tsx:29]: OK: #95

**Pull request merged**